BlueDBOnDisk/src/main/java/org/bluedb/disk/recovery/IndividualChange.java

```diff
@@ -18,6 +18,10 @@
 		return new IndividualChange<T>(key, null, value);
 	}
 
+	public static <T extends Serializable> IndividualChange<T> createDeleteChange(BlueKey key) {
+		return new IndividualChange<T>(key, null, null);

```

Comment:
The delete multiple has the current value as the first option and null as the second. I'm guessing this works fine for the batch delete, I just want to make sure

Reply:
It should, and should be tested